### ISSUE TITLE
when running migrate, settings.py is being used instead of local_settings.py

### ISSUE THREAD

**Issue by matikin9** [hackforla/adopt-civic-art#30]
I assumed the point of having a `local_settings.py` was so that django uses that file instead of `settings.py`, however when i follow the setup instructions to run the migrate, it seems to be using `settings.py` (and thus tries to get a google maps api key environment variable).

Am I mis-understanding the intended behavior, or is there a config setting somewhere I should change to point to `local_settings.py`?

![image](https://user-images.githubusercontent.com/1873072/28982457-ce574712-790a-11e7-8fe5-72f5742c8540.png)

**Comment by matikin9** [comment]
pinging @xtine

**Comment by xtine** [comment]
You're right, without setting an environment variable for `GOOGLE_MAPS_API_KEY`, it fails in settings looking for one. I just committed a fix for this (https://github.com/hackforla/adopt-civic-art/commit/4e95b22dd992e3a5e6b173715c63534009199c7d) that will bypass the need for an environment variable (will just be '' if no env var set), and set the one defined in `local_settings.py`.

Also in addition to this I also updated the behavior of password locking the site so if there is no env var password set it wont lock the site.

Pull latest, and if it works for you now, feel free to close the issue. Thanks for pointing this out!
